packages/lab/src/tag/Tag.css

```diff
@@ -0,0 +1,179 @@
+.saltTag {
+  height: calc(var(--salt-size-base) - var(--salt-spacing-100));
+  min-height: var(--salt-text-minHeight);
+  display: flex;
+  column-gap: var(--salt-spacing-50);
+  align-items: center;
+  padding: var(--salt-spacing-25) var(--salt-spacing-100);
+  font-family: var(--salt-text-fontFamily);
+  font-size: var(--salt-text-fontSize);
+  font-weight: var(--salt-text-fontWeight);
+  line-height: var(--salt-text-lineHeight);
+  overflow: hidden;
+  border-radius: var(--saltTag-borderRadius, var(--salt-palette-corner-strongest, 9999px));
+  width: fit-content;
+  min-width: max-content;
+  border-box: box-sizing;
+}
+.saltTag-primary {
+  background: var(--tag-primary-background, var(--salt-category-1-subtle-background));
+  color: var(--tag-primary-foreground, var(--salt-category-1-subtle-foreground));
+  border: var(--salt-size-border) var(--salt-selectable-borderStyle) transparent;
+}
+.saltTag-bordered {
+  border: var(--salt-size-border) var(--salt-selectable-borderStyle) var(--tag-primary-borderColor, var(--salt-category-1-subtle-borderColor));
```

Comment:
Is selectable-borderStyle the right token?

Reply:
```suggestion
  border: var(--salt-size-border) var(--salt-container-borderStyle) var(--tag-primary-borderColor, var(--salt-category-1-subtle-borderColor));
```

Reply:
this means it's missing from design spec, needs to flag